### PR TITLE
Adds coverage folder to gitignore

### PR DESCRIPTION
**Pull request by PenneyGadget**
@rrgayhart I added the coverage folder to the .gitignore - I noticed when I just pulled master that it pulled down some 370 lines of code in one of the files from that folder.

### CODE REVIEW

File: coverage/.resultset.json [hunk 1]
```diff
@@ -1,26 +1,26 @@
 {
```
Comment (rrgayhart) [inline coverage/.resultset.json]: Good catch! It looks like you need to remove this one file as well

Comment (PenneyGadget) [inline coverage/.resultset.json]: Which file?

Comment (rrgayhart) [inline coverage/.resultset.json]: This coverage/.resultset.json

Comment (rrgayhart) [inline coverage/.resultset.json]: You'll want to remove the coverage folder from the git cache:

"call `git rm --cached` on each of the files you want to remove from version control"

And then commit removing the files from source control!

**Pull request merged**